### ISSUE TITLE
HttpError: Resource not accessible by integration

### ISSUE THREAD

**Issue by GuillaumeFalourd** [Madrapps/add-reviewers#1]
Hello, I just got a `Error: HttpError: Resource not accessible by integration` when using the action:

<img width="479" alt="Screenshot 2021-10-14 at 08 55 26" src="https://user-images.githubusercontent.com/22433243/137312743-acbad657-6d7a-4df9-a0c5-15ce219ec84b.png">

You can find the workflow implementation here: https://github.com/GuillaumeFalourd/useful-actions/blob/main/.github/workflows/add-reviewers.yml

The workflow run with the error can also be found here: https://github.com/GuillaumeFalourd/useful-actions/runs/3894145396?check_suite_focus=true

Any idea why it happened? And how to resolve it?

**Comment by prince-chrismc** [comment]
This is probably from a public fork? IF so it's a well known issue... I wrote https://github.com/prince-chrismc/label-merge-conflicts-action#faq---how-do-i-fix-resource-not-accessible-by-integration which should help explain

TL;DR, replace `pull_request` with `pull_request_target`

**Comment by GuillaumeFalourd** [comment]
Hi! And thank you for the comment 😄 

I tried your suggestion, using `pull_request_target` instead of ` pull_request` as a workflow trigger.

The result is another error: `Error: Only pull request is supported, pull_request_target not supported.`

The PR was from the same repository, not from a FORK.

Workflow run: https://github.com/GuillaumeFalourd/useful-actions/runs/5310843867?check_suite_focus=true
